### PR TITLE
Add allowPrivilegeEscalation to kubectl describe psp

### LINKED ISSUE #57974: kubectl describe psp doesn't describe allowPrivilegeEscalation option
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug
/sig auth

**What happened**:
`kubectl describe psp` doesn't describe `allowPrivilegeEscalation` option.

**What you expected to happen**:
`allowPrivilegeEscalation` value should be present.

**How to reproduce it (as minimally and precisely as possible)**:
```console
$ kubectl describe psp restricted | grep -i Escalation
$
```

**Environment**:
```
Client Version: version.Info{Major:"", Minor:"", GitVersion:"v1.9.0-beta1", GitCommit:"3258431", GitTreeState:"clean", BuildDate:"2018-01-04T16:20:36Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10+", GitVersion:"v1.10.0-alpha.1.556+af78b9bac4d34e-dirty", GitCommit:"af78b9bac4d34eeec4adcf48f90845d5b325e081", GitTreeState:"dirty", BuildDate:"2018-01-08T14:52:54Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
```

### PR DESCRIPTION
**Pull request by alexcope**
**What this PR does / why we need it**:
Add allowPrivilegeEscalation to kubectl describe psp

**Which issue(s) this PR fixes**
Fixes #57974 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by liggitt** [comment]
Can you include sample output?

**Comment by mikedanese** [comment]
/approve

LGTM, pending sample output.

**Comment by alexcope** [comment]
Here is sample output:
```
$ kubectl describe psp
Name:  gce.privileged

Settings:
  Allow Privileged:                       true
  Allow Privilege Escalation:             true
  Default Add Capabilities:               <none>
  Required Drop Capabilities:             <none>
  Allowed Capabilities:                   *
  Allowed Volume Types:                   *
  Allow Host Network:                     true
  Allow Host Ports:                       0-65535
  Allow Host PID:                         true
  Allow Host IPC:                         true
  Read Only Root Filesystem:              false
  SELinux Context Strategy: RunAsAny      
    User:                                 <none>
    Role:                                 <none>
    Type:                                 <none>
    Level:                                <none>
  Run As User Strategy: RunAsAny          
    Ranges:                               <none>
  FSGroup Strategy: RunAsAny              
    Ranges:                               <none>
  Supplemental Groups Strategy: RunAsAny  
    Ranges:                               <none>

```

**Comment by liggitt** [comment]
thanks
/lgtm

File: pkg/printers/internalversion/describe.go [hunk 1]
```diff
@@ -3392,6 +3392,7 @@ func describePodSecurityPolicy(psp *extensions.PodSecurityPolicy) (string, error
 		w.Write(LEVEL_0, "\nSettings:\n")
 
 		w.Write(LEVEL_1, "Allow Privileged:\t%t\n", psp.Spec.Privileged)
+		w.Write(LEVEL_1, "Allow Privilege Escalation:\t%v\n", psp.Spec.AllowPrivilegeEscalation)
```
Comment (php-coder) [inline pkg/printers/internalversion/describe.go]: @liggitt Do you remember, is it possible that this value will be `nil` (when the value of the `DefaultAllowPrivilegeEscalation` is used)?

**Comment by liggitt** [comment]
Defaulting turns a nil value into true

**Comment by php-coder** [comment]
BTW this PR could also update the test `TestDescribePodSecurityPolicy`.

**Pull request merged**